### PR TITLE
Add Docker RHEL

### PR DESCRIPTION
**Pull request by sara-02**
Merge after merging #15

### CODE REVIEW

File: Dockerfile.rhel [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# base image source https://github.com/rhdt/EL-Dockerfiles/blob/master/base/python3/Dockerfile
+
+FROM prod.registry.devshift.net/osio-prod/base/python3:latest
+
+LABEL maintainer="Sarah Masud <smasud@redhat.com>"
+
+COPY ./requirements.txt /requirements.txt
+RUN pip3 install -r /requirements.txt && rm /requirements.txt
+
+COPY ./src /src
```
Comment (msrb) [inline Dockerfile.rhel]: I believe you will need to update this dockerfile once swagger PR is merged.

Comment (sara-02) [reply Dockerfile.rhel]: Sure, I will update the Swagger PR as well.

**Comment by msrb** [comment]
@sara-02 LGTM, feel free to merge :wink:

**Comment by sara-02** [comment]
@msrb Thanks, merging this now.

**Pull request merged**